### PR TITLE
Fix GitHub Actions service

### PR DESCRIPTION
**Pull request by thomasrockhu**

### CODE REVIEW

**Comment by thomasrockhu** [comment]
yeah, it was just `black`, made a fix and should pass

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     packages=["codecov"],
     include_package_data=True,
     zip_safe=True,
-    install_requires=["requests>=2.7.9", "coverage"],
+    install_requires=["requests>=2.7.9", "coverage==4.5.4"],
```
Comment (Maks3w) [inline setup.py:43]: This change breaks the detection in other systems

```
==> Collecting reports
 
    Generating coverage xml reports for Python
 
    Error running `['/usr/local/bin/python', '-m', 'coverage', 'xml', '-i']`: Command '['/usr/local/bin/python', '-m', 'coverage', 'xml', '-i']' returned non-zero exit status 1.
 
    - Ignored: [Errno 2] No such file or directory: '/root/app/src/coverage.xml'
 
Error: No coverage report found

With coverage 5.1 it was working fine
 
Tip: See all example repositories: https://github.com/codecov?query=example
 
```

Comment (thomasrockhu) [reply setup.py:43]: Thanks @Maks3w, I'll make a fix for this and push a new version

Comment (thomasrockhu) [reply setup.py:43]: @Maks3w, I just pushed `2.1.7`, let me know if that fixes your issue

Comment (Maks3w) [reply setup.py:43]: It works fine again with 2.17. Thx

**Pull request merged**